pygmt/x2sys.py

```diff
@@ -267,10 +269,10 @@ def x2sys_cross(tracks=None, outfile=None, **kwargs):
         not calculate a COE. Specify:
 
         - **l** sets lower speed [Default is 0].
-        - **u** sets upper speed [Default is Infinity].
-        - **h** does not limit the speed but sets a lower speed below which \
-        headings will not be computed (i.e., set to NaN) [Default calculates \
-        headings regardless of speed].
+        - **u** sets upper speed [Default is infinity].
+        - **h** does not limit the speed but sets a lower speed below which
+          headings will not be computed (i.e., set to NaN) [Default
+          calculates headings regardless of speed].
 
         For example, you can use ``speed=["l0", "u10", "h5"]`` to set a lower
         speed of 0, upper speed of 10, and disable heading calculations for
```

Comment:
Change `pandas.DataFrame` to <code>:class:\`pandas.DataFrame\`</code>

Reply:
Why is `pandas.DataFrame` automatically linked under the Parameters section but not under Returns?

Reply:
Good question.

Reply:
Ok, I see Meghan opened up issue #937 so resolving this.